packages/react-components/src/components/EndCallButton.tsx

```diff
@@ -95,7 +95,8 @@ const darkThemeCallButtonStyles = {
 const lightThemeCallButtonStyles = {
   root: {
     color: lightTheme.callingPalette.iconWhite,
-    background: lightTheme.callingPalette.callRed
+    background: lightTheme.callingPalette.callRed,
+    ':focus::after': { outlineColor: `${lightTheme.callingPalette.iconWhite} !important` } // added !important to avoid override by FluentUI button styles
```

Comment:
nit. Its harder to read these values. Can we treat this "object" like any other "object" and when we have curly braces, have the content on the next line?

```
'focus::after': {
   outlineColor: `${lightTheme.callingPalette.iconWhite} !important`
}
```